### ISSUE TITLE
Regressions in 2.2 update

### ISSUE THREAD

**Issue by Leland** [SnowdogApps/magento2-theme-blank-sass#164]
Looks like #161 introduced some discrepancies. Chiefly, the removal of `!default` flags.

E.g https://github.com/SnowdogApps/magento2-theme-blank-sass/blob/34827e76e74ca12b21e510056d8e8da2b55cd563/Magento_Checkout/styles/module/checkout/_shipping.scss

https://github.com/SnowdogApps/magento2-theme-blank-sass/blob/34827e76e74ca12b21e510056d8e8da2b55cd563/Magento_Checkout/styles/module/checkout/_sidebar-shipping-information.scss

https://github.com/SnowdogApps/magento2-theme-blank-sass/blob/34827e76e74ca12b21e510056d8e8da2b55cd563/Magento_Checkout/styles/module/checkout/_tooltip.scss

...and many other files. Ideally, any variable definitions should end with a default flag.

Also, not sure about the usage of `lib-css` – it looks like that's used primarily to output vendor prefixed values and Frontools' autoprefixer takes care of that by default.

**Comment by Igloczek** [comment]
About the `!default` flags - you are right, feel free to submit PR or just wait a week or two for changes on our side.

About `lib-css` - no, is not related just to prefixes, it will let us to not add any properties, while variable have a `false` value. For more detail please check the source code.

**Comment by Leland** [comment]
> please don't complain about our work in that form, it's not nice for maintainers

Greatly appreciate the work you're doing here! Do apologize if there was a bad tone in my opened issue; can promise that there isn't one in my intent. I'm a contributor to this repository and may be able to fix this if you don't get to it first. Cheers.

**Comment by BenGlassAtlas** [comment]
I have noticed one issue where variables are set to `false` the word `false` is included in the CSS.

**Variable declaration**

```scss
File: vendor/snowdog/theme-blank-sass/Magento_Theme/styles/_module.scss:38

$footer__background-color                  : false;
```

**Usage**

```scss
File: vendor/snowdog/theme-blank-sass/Magento_Theme/styles/_module.scss:149

.page-footer {
    @include lib-css(background-color, $footer__background-color);
    margin-top: auto;
}
```

**Output**

```css
.page-footer {
  background-color: false;
  margin-top: auto; }
```

**Comment by Igloczek** [comment]
@BenSpace48 weird, `lib-css` mixin should prevent that thing...

**Comment by talalus** [comment]
@BenSpace48 BenSpace48 I tested `lib-css` with snowdog/theme-blank-sass:1.0.2 and snowdog/frontools:1.5.8 and I got the expected result 
```
.page-footer {
  margin-top: auto; }
```
which version of frontool and theme are you using?

**Comment by BenGlassAtlas** [comment]
The same versions weirdly, maybe it's related to my customisations to the theme (although we haven't overwritten `vendor/snowdog/theme-blank-sass/styles/vendor/magento-ui/_utilities.scss`. 

I'm very pushed for time at the minute so I can't check if the same thing happens in the base sass theme, I'll try and confirm when I get time.

**Comment by Igloczek** [comment]
I'm closing this issue, b/c `!default` flags are already added on `develop` branch and will be released with 1.1.0.

`false` issue didn't get any follow-up, so I believe it just works now.
